### PR TITLE
fix(curriculum): changed description for anchor tag in learn-html-by-building-a-cat-photo-app

### LINKED ISSUE #50578: Unclear instructions in step 64 of building a cat photo app
### Describe the Issue

It might be just me being a bit confused with the instruction, but someone from the forum had this issue and it also took me a bit of time to get the instruction right as well

So it says, "Make the text `freeCodeCamp.org` into a link by enclosing it in an anchor (a) element. The href attribute should be set to https://www.freecodecamp.org." I think it would be better if it read something more along the lines of "Replace the text `freeCodeCamp.org` in the p tag with a link, by enclosing it in an anchor (a) element. The href attribute should be set to https://www.freecodecamp.org."

I was tempted to think that I needed an extra anchor tag outside the p tag with the text `freeCodeCamp.org`, and the hint given to me when the test failed only made me more confused because it said "Your anchor (a) element should be nested within the footer element. Make sure to add an opening tag and closing tag for the anchor (a) element.". Even though I was wrong to create the anchor tag outside of the p tag, it was enclosed within the footer element.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-64

### Your code

```

<p> No Copyright - freeCodeCamp.org </p>
 <a href="https://www.freecodecamp.org">
      freeCodeCamp.org
  </a>

```
This is what I did, and it failed.

### Expected behavior

I expected the instruction to be a little bit more precise, or maybe a little more improvement on the hint.

### Screenshots

_No response_

### System

- Device: Laptop
- OS: Windows 11
- Browser: Chrome

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by midikaif**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #50578

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804e9.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ dashedName: step-64
 
 # --description--
 
-Make the text `freeCodeCamp.org` into a link by enclosing it in an anchor (`a`) element. The `href` attribute should be set to `https://www.freecodecamp.org`.
+Enclose below text `freeCodeCamp.org` with an anchor (`a`) element to make it a link and set the `href` attribute of `a` element to `https://www.freecodecamp.org`.
```
Comment (bbsmooth) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804e9.md:10]:
```suggestion
Turn the existing `freeCodeCamp.org` text into a link by enclosing it in an anchor (`a`) element. The `href` attribute should be set to `https://www.freecodecamp.org`."
```

I think your new wording is slightly awkward. My suggestion is @moT01 's wording from the issue.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804e9.md [hunk 2]
```diff
@@ -7,11 +7,11 @@ dashedName: step-64
...
 
 # --hints--
 
-Your anchor (`a`) element should be nested within the `footer` element. Make sure to add an opening tag and closing tag for the anchor (`a`) element.
+Your anchor (`a`) element should be nested within the `p` element. Make sure to add an opening tag and closing tag for the anchor (`a`) element.
```
Comment (bbsmooth) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804e9.md:14]:
```suggestion
Your anchor (`a`) element should be nested within the `p` element inside the `footer`. Make sure to add an opening tag and closing tag for the anchor (`a`) element.
```

There are multiple `p` elements in the HTML so we should specify which one.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804e9.md (hunk 1 above)
Comment (midikaif) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804e9.md:10]: ok. I'll add @moT01 's wording.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804e9.md [hunk 3]
```diff
@@ -7,11 +7,11 @@ dashedName: step-64
 
 # --description--
 
-Make the text `freeCodeCamp.org` into a link by enclosing it in an anchor (`a`) element. The `href` attribute should be set to `https://www.freecodecamp.org`.
+Turn the existing `freeCodeCamp.org` text into a link by enclosing it in an anchor (`a`) element. The `href` attribute should be set to `https://www.freecodecamp.org`."
```
[the remark this answers is not in the document]
Comment (midikaif) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804e9.md:10]: Sorry sorry! I didn't see that

**Review by huyenltnguyen — APPROVED** [review]
LGTM 👍 Thank you @midikaif!

**Review by gikf — APPROVED** [review]

**Pull request merged**